src/Microsoft.DotNet.Interactive/Commands/UpdateDisplayedValue.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.DotNet.Interactive.Commands
 {
     public class UpdateDisplayedValue : KernelCommand
     {
-        public UpdateDisplayedValue(FormattedValue formattedValue, string valueId)
+        public UpdateDisplayedValue(FormattedValue formattedValue, string valueId, string targetKernelName = null, string theme = null)
```

Comment:
We shouldn't make any changes to events or commands as this will have an impact on our public API contract.